plugins/env/app/models/environment_variable.rb

```diff
@@ -5,7 +5,9 @@ class EnvironmentVariable < ActiveRecord::Base
   validates :scope_type, inclusion: ["Environment", "DeployGroup", nil]
 
   class << self
-    def env(project, deploy_group)
+    # preview parameter can be used to not raise an error,
+    # but return a value with a helpful message
+    def env(project, deploy_group, preview: false)
```

Comment:
it does not, I'm going to use it in the consul reader to either blow up or show a warning